pkg/schedule/schedulers/hot_region_test.go

```diff
@@ -527,25 +534,12 @@ func checkHotWriteRegionScheduleByteRateOnly(re *require.Assertions, enablePlace
 	clearPendingInfluence(hb.(*hotScheduler))
 	tc.SetHotRegionScheduleLimit(int(opt.GetHotRegionScheduleLimit()))
 
-	for i := 0; i < 20; i++ {
```

Comment:
This part is duplicated with L500-L522. But this branch, which tests the transfer leader, is not reached, so it is not updated before.